tests/test_teammate_allowlist.sh

```
@@ -0,0 +1,84 @@
+}
+
+fail() {
+  FAILED=$((FAILED + 1))
+  TOTAL=$((TOTAL + 1))
+  echo "  FAIL: $1"
+}
+
+assert_allow_contains() {
+  local settings_path="$1"
+  local pattern="$2"
+  local desc="allowlist: ${settings_path#$REPO_ROOT/} に $pattern"
```

Shellcheck (0.11.0)

[info] Expansions inside ${..} need to be quoted separately, otherwise they match as patterns.

(SC2295)